pages/docs/tutorials/spring-boot-restful.md

```diff
@@ -19,26 +19,26 @@ Note that all classes in this tutorial are in the `org.jetbrains.kotlin.demo` pa
 ### Defining the project and dependencies
 {{ site.text_using_gradle }}
 
-The Gradle file is pretty much standard for Spring Boot. The only difference is the structure layout for source folders for Kotlin, and the required Kotlin dependencies
+The Gradle file is pretty much standard for Spring Boot. The only differences are the structure layout for source folders for Kotlin, the required Kotlin dependencies and the *kotlin-spring* Gradle plugin which prevents using *open* modifier at class and member function level for Spring beans proxified with CGLIB.
```

Comment:
As far as I understand *kotlin-spring* plugin does not _prevent_ using `open`, quite the opposite—it automatically puts it in place during compilation.